src/libnvme.map

```diff
@@ -1,5 +1,10 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+LIBNVME_1_6 {
```

Comment:
The current git master is still v1.5 (unreleased)

Reply:
Thansk for your comment. Just fixed the patch to use the map file v1.5 definition.